client/blocks/inline-help/admin-sections.js

```diff
@@ -0,0 +1,381 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { memoize } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+
+/**
+ * Returns admin section items with site-based urls.
+ *
+ * @param {String} siteSlug - The current site slug.
+ * @returns {Array}           An array of admin sections with site-specific URLs.
+ */
+export const adminSections = memoize( ( siteSlug ) => [
+	{
+		title: translate( 'Add a new domain' ),
+		description: translate(
+			'Set up your domain whether it’s registered with WordPress.com or elsewhere.'
```

Comment:
```suggestion
			'Create your domain with WordPress.com.'
```

The English wasn't quite right. Also it's "adding" a domain not checking where it's registered right?

Reply:
No idea :-). I've copied&pasted all these texts from https://github.com/Automattic/wp-calypso/pull/31903 and also updated according to feedback collected here See 8-gh-dotcom-manage.

Reply:
Ah I see. I thought it was from somewhere else. 

Ok leave as is then. It didn't feel fluent to me, but having re-read it doesn't seem that bad.